core/DataAccess/Model.php

```diff
@@ -632,10 +632,10 @@ public function getArchiveIdsForSegments($archiveTableName, array $segments, $ol
     private function getDeletedSegmentWhereClause(array $segment)
     {
         $idSite = (int)$segment['enable_only_idsite'];
-        $segmentHash = $segment['hash'];
+        $segmentHash = $segment['hash'] ?? '';
```

Comment:
@diosmosis Do you have a quick idea why the hash might be unset here? Haven't had a deeper look at the code where that might come from.

Reply:
@sgiehl looks like a bug, `hash` isn't selected in SegmentEditor\Model::getSegmentsDeletedSince()

Reply:
good catch. Updated that method 👍